src/Raven.Studio/typescript/viewmodels/database/patch/patch.ts

```diff
@@ -205,7 +208,16 @@ class patch extends viewModelBase {
 
         this.loadLastQuery();
 
-        return $.when<any>(this.fetchAllIndexes(this.activeDatabase()), this.savedPatches.loadAll(this.activeDatabase()));
+        const studioDatabaseConfigTask = new getDatabaseStudioConfigurationCommand(this.activeDatabase())
```

Comment:
IIRC studio confgiratuoin is loaded when we change db and it is persistent in memory. Please have a look at usages.

Reply:
done